webpack.config.js

```diff
@@ -65,79 +65,89 @@ const config = {
         // node_modules. That would take a long time.
         exclude: /node_modules/,
         // use the babel loader
-        loader: 'babel-loader',
+        loader: "babel-loader",
```

Comment:
I'm not sure why your autoformatter changed these all to double quotes. Can you see if you can fix this back to single quotes for all files?

Reply:
I honestly have no idea how